### PR TITLE
356: CSS selectors as per coding standards

### PR DESCRIPTION
**Pull request by anevins12**
#356 Resolves the selector on new lines coding standard

### CODE REVIEW

File: style-rtl.css [hunk 1]
```diff
@@ -2790,7 +2809,7 @@ body.page .main-navigation {
   cursor: pointer;
 }
 
-.entry-content .wp-block-button .wp-block-button__link:hover, .entry-content .wp-block-button .wp-block-button__link:focus {
+.entry-content .wp-block-button .wp-block-button__link:focus, .entry-content .wp-block-button .wp-block-button__link:hover {
```
[the remark this answers is not in the document]
Comment (anevins12) [reply style-rtl.css]: Hi @ntwb , I'm not sure how to achieve that, as the source scss file already has your recommendation.

[the remark this answers is not in the document]
Comment (ntwb) [reply style-rtl.css]: Right, yeah as I made the "suggested changes" inline here in the PR (which is a new GutHub feature) those changes will be seen as already made.

**Comment by ntwb** [comment]
@anevins12 If you can fix the conflict noted above it should be good to merge 👍

**Comment by anevins12** [comment]
Can you double check that I chose the right conflict resolution

File: style.css [hunk 2]
```diff
@@ -3219,7 +3239,7 @@ body.page .main-navigation {
   cursor: pointer;
 }
 
-.entry-content .wp-block-file .wp-block-file__button:hover, .entry-content .wp-block-file .wp-block-file__button:focus {
+.entry-content .wp-block-file .wp-block-file__button:focus, .entry-content .wp-block-file .wp-block-file__button:hover {
```
[the remark this answers is not in the document]
Comment (allancole) [reply style.css]: This rules are now set separately in the Sass file. 
See: https://github.com/WordPress/twentynineteen/pull/372/commits/968c3528c81ae0ad83e7af9cf31f1ceca14dbaf0

**Pull request merged**